go/samples/http/main.go

```diff
@@ -1,21 +1,24 @@
 package main
```

Comment:
The goal of this PR was to make the existing example work on multiple databases. DIdn't really think of CUJs. However, since we are quite close to the bug bash, I might as well add CUJs to this PR